pyugrid/read_netcdf.py

```diff
@@ -23,7 +25,7 @@ def find_mesh_names( nc ):
     NOTE: checks for 2-d topology_dimension
     """
     mesh_names = []
-    for varname in nc.variables.iterkeys():
```

Comment:
The trick here is that we'll get the full list, rather than an iterator in py2 -- maybe we don't care?

We could use the six module for this, but that's kind of ugly -- maybe wait 'till we really need it?